libobs/obs-interaction.h

```diff
@@ -41,6 +41,19 @@ enum obs_mouse_button_type {
 	MOUSE_RIGHT
 };
 
+enum obs_navigation_key_flags
+{
```

Comment:
Style: Place this curly brace at the end of the preceding line with a space before it.